analysis/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+import json
+import jwt
+import requests
+import numpy
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+from django.conf import settings
+
+from users.models import User
+from users.utils import login_decorator
+from movie.models import Movie
+from .models import Star
+
+class StarratingView(View):
+    @login_decorator
+    def get(self, request):
+        if request.GET.get('id', None) != None:
```

Comment:
```suggestion
         ???_id  = request.GET.get('id', None) 
        if ???_id:
```
위의 id는 어떤 아이디 일까요? 영화 id 인가요? 
제대로 담기고 None이 아니라면 True 이므로 != None 할 필요 없습니다.